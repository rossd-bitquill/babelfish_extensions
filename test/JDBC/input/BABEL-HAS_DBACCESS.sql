DROP DATABASE IF EXISTS TestDB;
CREATE DATABASE TestDB;
SELECT HAS_DBACCESS('TestDB');
GO

SELECT HAS_DBACCESS('does_not_exist');
GO

DROP DATABASE TestDB;
SELECT HAS_DBACCESS('TestDB');
GO
