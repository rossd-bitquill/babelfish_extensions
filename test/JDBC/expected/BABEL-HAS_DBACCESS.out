DROP DATABASE IF EXISTS TestDB;
CREATE DATABASE TestDB;
SELECT HAS_DBACCESS('TestDB');
GO
~~START~~
int
1
~~END~~


SELECT HAS_DBACCESS('does_not_exist');
GO
~~START~~
int
1
~~END~~


DROP DATABASE TestDB;
SELECT HAS_DBACCESS('TestDB');
GO
~~START~~
int
1
~~END~~

